ignite/pkg/cosmosgen/generate_typescript.go

```diff
@@ -54,7 +60,14 @@ func newTSGenerator(g *generator) *tsGenerator {
 	}
 
 	if !tsg.isLocalProto {
-		log.Printf("No '%s' binary found in PATH, using remote buf plugin for Typescript generation. %s\n", protocGenTSProtoBin, msgBufAuth)
+		if os.Getenv(bufTokenEnvName) == "" {
+			token, err := buf.FetchToken()
+			if err != nil {
+				log.Printf("No '%s' binary found in PATH, using remote buf plugin for Typescript generation. %s\n", protocGenTSProtoBin, msgBufAuth)
+			} else {
+				os.Setenv(bufTokenEnvName, token)
```

Comment:
Maybe we should delete the env token after we run the command, for the user don't keep use our tokens for other buf request